blockly-core/core/ui/block.js

```diff
@@ -543,7 +543,8 @@ Blockly.Block.prototype.onMouseDown_ = function(e) {
   // appears to cause IE to stop propogating the event. We don't want this to
   // happen when we're clicking on an input target, and dont really want
   // to select in that case anyways.
-  if (e.target.getAttribute('class') !== 'inputClickTarget') {
+  var targetClass = e.target.getAttribute && e.target.getAttribute('class');
+  if (targetClass !== 'inputClickTarget') {
```

Comment:
Yeah, my comment is wrong. We call select UNLESS we're able to getAttribute and it's an inputClickTarget. The code rather than my description is the desired behavior luckily :)